WORKSPACE.bzlmod

```diff
@@ -195,3 +195,7 @@ browser_repositories(chromium = True)
 load("@io_bazel_rules_webtesting//web:go_repositories.bzl", web_test_go_repositories = "go_repositories")
 
 web_test_go_repositories()
+
+register_toolchains(
+    "@buildbuddy_toolchain//:ubuntu_cc_toolchain",
+)
```

Comment:
ditto: move this to `MODULE.bazel`

Reply:
Can't; we need the `@buildbuddy_toolchain` repo created by the `buildbuddy` call, and we can't `load` the `buildbuddy` call in `MODULE.bazel`. We'd need to make it an extension, I think? Which I have not looked into how to do.

Reply:
Oh you are right.

Let's use your approach for now. I will add support for bzlmod in our toolchain and change this part later.